homeassistant/components/ifttt/alarm_control_panel.py

```diff
@@ -153,6 +173,11 @@ def code_format(self):
             return FORMAT_NUMBER
         return FORMAT_TEXT
 
+    @property
+    def code_arm_required(self):
```

Comment:
My impression is this is not currently used, and it seemed like this change was standalone and would not be built on.  Want to remove this?  If you plan to use it in a follow up PR, no problem -- just thought I would ask.

Reply:
I remove lines 176 to 180

Thank you